src/components/Route/CFunctionsStepper.vue

```diff
@@ -0,0 +1,331 @@
+              'example': 'request.limit = 1',
+            },
+          ],
+          'label': 'Rate limiting validator',
+          'params': [
+            {
+              'label': 'expr',
+              'type': 'expr',
+              'options': {},
+            },
+          ],
+        },
+        {
+          'ref': 'validatorSchema',
+          'kind': 'validator',
+          'step': 1,
+          'label': 'Schema validator',
+          'params': [
+            {
+              'label': 'schema',
+              'type': 'textarea',
+              'options': {
+                'rteSettingsExample': '',
+              },
+            },
+          ],
+        },
+        {
+          'ref': 'validatorContentLength',
+          'kind': 'validator',
+          'step': 1,
+          'label': 'Content length validator',
+          'scope': [
+            {
+              'type': 'string',
+              'label': 'request',
+              'example': 'request.length > 1024',
+            },
+          ],
+          'params': [
+            {
+              'label': 'expr',
+              'type': 'expr',
+              'options': {
+                'placeholder': 'expression placeholder here...',
+              },
+            },
+          ],
+        },
+        {
+          'ref': 'processerWorkflow',
+          'kind': 'processer',
+          'step': 2,
+          'label': 'Workflow processer',
+          'params': [
+            {
+              'label': 'workflow',
+              'type': 'workflow',
+              'options': {
+                'options': [],
+              },
+            },
+          ],
+        },
+        {
+          'ref': 'expediterRedirection',
+          'kind': 'expediter',
+          'step': 3,
+          'label': 'Redirection expediter',
+          'params': [
+            {
+              'label': 'location',
+              'type': 'string',
+              'options': [],
+            },
+            {
+              'label': 'status',
+              'type': 'status',
+              'options': {
+                'options': [
+                  'http 500',
+                  'http 404',
+                ],
+              },
+            },
+          ],
+        },
+        {
+          'ref': 'expediterJsonEncoder - TODO',
+          'kind': 'expediter',
+          'step': 3,
+          'label': 'Json encoder expediter',
+        },
+        {
+          'ref': 'expediterCompresser',
+          'kind': 'expediter',
+          'step': 3,
+          'label': 'Gzip compresser expediter',
+          'params': [
+            {
+              'label': 'setHeaders',
+              'type': 'bool',
+            },
+            {
+              'label': 'level',
+              'type': 'int',
+            },
+          ],
+        },
+      ]
+    },
+    getRouteFunctions () {
+      // Remove step functions from steps
+      // Use routeFunctions
+
+    },
+  },
+}
+</script>
+
+<style scoped>
```

Comment:
@MarioBur can you please handle this with Bootstrap columns? Don't forget everything we do has to be mobile responsive and this percentage approach won't meet that requirement.